### PR TITLE
This commit add in species upload capabilities

### PR DESCRIPTION
**Pull request by ahinz**

### CODE REVIEW

File: api/test_utils.py [hunk 1]
```diff
@@ -196,9 +196,15 @@ def local_render_to_response(*args, **kwargs):
     agn1.save()
     agn2.save()
 
-    s1 = Species(symbol="s1",genus="testus1",species="specieius1")
-    s2 = Species(symbol="s2",genus="testus2",species="specieius2")
-    s3 = Species(symbol="s3",genus="testus2",species="specieius3")
+    s1 = Species(symbol="s1",genus="testus1",species="specieius1",
```
Comment (steventlamb) [inline api/test_utils.py:199]: "specieius"? Is this intentional? occurs in several places.

File: importer/models.py [hunk 2]
```diff
@@ -238,9 +270,11 @@ def safe_bool(self, fld):
         """ Returns a tuple of (success, bool value) """
         v = self.datadict.get(fld, '').lower()
```
Comment (steventlamb) [inline importer/models.py:271]: a longer variable name would make this a bit easier to understand.

**Comment by steventlamb** [comment]
Consider amending the commit message to something like "Add in species upload capabilities"

File: api/test_utils.py (hunk 1 above)
Comment (ahinz) [inline api/test_utils.py:199]: yeah- it's Latin.... duh

**Pull request merged**